esp/src/package.json

```diff
@@ -40,6 +40,8 @@
     "@hpcc-js/other": "2.13.12",
     "@hpcc-js/tree": "2.9.1",
     "@material-ui/core": "4.8.3",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.42",
```

Comment:
Is this needed?

Reply:
I was adding a feature that is not included in the first version of this. So, I suppose we can remove until it is needed.